lightautoml/pipelines/selection/base.py

```diff
@@ -212,11 +211,14 @@ def select(self, dataset: LAMLDataset) -> LAMLDataset:
 
     def map_raw_feature_importances(self, raw_importances: Series):
         """Calculate input feature importances.
+
         Calculated as sum of importances on different levels of pipeline.
 
         Args:
             raw_importances: Importances of output features.
 
+        # noqa: DAR201
```

Comment:
Does it work?

Reply:
Yes, so all errors in this line are disabled, but the error code is specified for users